src/components/icon/icon.stories.tsx

```diff
@@ -0,0 +1,278 @@
+  },
+  {
+    iconId: "arrow-up"
+  },
+  {
+    iconId: "pause-circle-outline"
+  },
+  {
+    iconId: "pause-circle"
+  },
+  {
+    iconId: "play-circle-outline"
+  },
+  {
+    iconId: "play-circle"
+  },
+  {
+    iconId: "arrow-back-outline"
+  },
+  {
+    iconId: "arrow-forward-outline"
+  },
+  {
+    iconId: "checkmark"
+  },
+  {
+    iconId: "done-all"
+  },
+  {
+    iconId: "arrow-ios-back"
+  },
+  {
+    iconId: "arrow-ios-forward"
+  },
+  {
+    iconId: "expand"
+  },
+  {
+    iconId: "close"
+  },
+  {
+    iconId: "plus-circle-outline"
+  },
+  {
+    iconId: "plus-circle"
+  },
+  {
+    iconId: "image-outline"
+  },
+  {
+    iconId: "image"
+  },
+  {
+    iconId: "search-outline"
+  },
+  {
+    iconId: "block"
+  },
+  {
+    iconId: "menu-outline"
+  },
+  {
+    iconId: "settings-outline"
+  },
+  {
+    iconId: "settings"
+  },
+  {
+    iconId: "bell-outline"
+  },
+  {
+    iconId: "bell-fill"
+  },
+  {
+    iconId: "color-palette-outline"
+  },
+  {
+    iconId: "layers-outline"
+  },
+  {
+    iconId: "layers"
+  },
+  {
+    iconId: "eye-off-outline"
+  },
+  {
+    iconId: "eye-off"
+  },
+  {
+    iconId: "eye-outline"
+  },
+  {
+    iconId: "eye"
+  },
+  {
+    iconId: "paper-plane"
+  },
+  {
+    iconId: "more-horizontal"
+  },
+  {
+    iconId: "massage-circle-outline"
+  },
+  {
+    iconId: "massage-circle"
+  },
+  {
+    iconId: "person-remove-outline"
+  },
+  {
+    iconId: "person-remove"
+  },
+  {
+    iconId: "person-add-outline"
+  },
+  {
+    iconId: "person-add"
+  },
+  {
+    iconId: "mic-outline"
+  },
+  {
+    iconId: "mic"
+  },
+  {
+    iconId: "copy-outline"
+  },
+  {
+    iconId: "copy"
+  },
+  {
+    iconId: "pin-outline"
+  },
+  {
+    iconId: "pin"
+  },
+  {
+    iconId: "heart-outline"
+  },
+  {
+    iconId: "heart"
+  },
+  {
+    iconId: "email-outline"
+  },
+  {
+    iconId: "email"
+  },
+  {
+    iconId: "trash-outline"
+  },
+  {
+    iconId: "trash"
+  },
+  {
+    iconId: "maximize-outline"
+  },
+  {
+    iconId: "maximize"
+  },
+  {
+    iconId: "edit-outline"
+  },
+  {
+    iconId: "edit"
+  },
+  {
+    iconId: "credit-card-outline"
+  },
+  {
+    iconId: "credit-card"
+  },
+  {
+    iconId: "github"
+  },
+  {
+    iconId: "recaptcha"
+  },
+  {
+    iconId: "facebook"
+  },
+  {
+    iconId: "google"
+  },
+  {
+    iconId: "paypal"
+  },
+  {
+    iconId: "stripe"
+  },
+  {
+    iconId: "paid"
+  },
+  {
+    iconId: "united-kingdom-flag"
+  },
+  {
+    iconId: "russia-flag"
+  }
+]
+
+export const Default: Story = {
+  render: () => (
+    <div
+      style={{
+        alignItems: "center",
+        display: "flex",
+        flexWrap: "wrap",
+        gap: "5px",
+        justifyContent: "space-around",
+        margin: "auto",
+        padding: "10%",
+        width: "50%"
+      }}
+    >
+      {iconsItems.map(i => {
+        return (
+          <Icon height={"34"} iconId={i.iconId} key={i.iconId} viewBox={"0 0 34 34"} width={"34"} />
```

Comment:
Could you please explain why you add custom values for viewBox, width and height? I guess default values is enough for preview in storybook